src/store/reducer/boardReaction.ts

```diff
@@ -0,0 +1,14 @@
+import {Action, ReduxAction} from "store/action";
+import {BoardReactionState} from "types/reaction";
+
+// eslint-disable-next-line @typescript-eslint/default-param-last
+export const boardReactionReducer = (state: BoardReactionState = [], action: ReduxAction): BoardReactionState => {
```

Comment:
why do you need the default value for state here? Can it, and the eslint-disable, be removed?

Reply:
yes, the default value seems to be required, but my IDE didn't like order of the params; state should be last in its opinion. I disabled the inspection now though.